### PR TITLE
[13.0][ADD] project_stock_product_set: New addon

### PR DESCRIPTION
**Pull request by victoralmau**
This module allows to use product sets in tasks.

Locked by:
- [x] `project_stock` https://github.com/OCA/project/pull/905

Please @pedrobaeza and @sergio-teruel can you review it?

@Tecnativa TT34555

### CODE REVIEW

**Comment by victoralmau** [comment]
Now, this is ready for review.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
This one should be called `project_stock_product_set`

**Comment by pedrobaeza** [comment]
It's red.

**Comment by victoralmau** [comment]
Rebase done, I think it's ready to merge.

File: project_stock_product_set/README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+#. Go to *Projects -> All task* and edit a task that allows set stock moves.
```
Comment (pedrobaeza) [inline project_stock_product_set/README.rst]: Indicate in CONFIGURE.rst what "allows to set stock moves" means.

```suggestion
#. Go to *Projects -> All tasks* and edit a task that allows to set stock moves.
```

File: project_stock_product_set/README.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+#. Go to *Add set* button and select some product set and click on the *Add set* button.
+#. Move lines have been created for each product in the *Stock Info* tab.
```
Comment (pedrobaeza) [inline project_stock_product_set/README.rst]:
```suggestion
#. Lines for each product of the set have been created in the *Stock Info* tab.
```

File: project_stock_product_set/views/project_task_view.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <field name="groups_id" eval="[(4, ref('stock.group_stock_user'))]" />
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
+                <button
+                    class="btn-primary"
+                    type="action"
+                    name="%(act_open_wizard_product_set_add_from_task)d"
+                    icon="fa-cubes"
+                    attrs="{'invisible': [('use_stock_moves','=',False)]}"
+                    groups="base.group_user"
```
Comment (pedrobaeza) [inline project_stock_product_set/views/project_task_view.xml]: Why this group?

**Review by pedrobaeza — APPROVED** [review]

File: project_stock_product_set/views/project_task_view.xml (hunk 3 above)
Comment (victoralmau) [reply project_stock_product_set/views/project_task_view.xml]: For similarity to https://github.com/OCA/sale-workflow/blob/13.0/sale_product_set/views/sale_order.xml#L15 was added but not necessary.

**Review by ernestotejeda — APPROVED** [review]

**Pull request merged**